models/ed/R/met2model.ED2.R

```diff
@@ -267,7 +267,7 @@ for(year in start_year:end_year) {
 
   ## write DRIVER file
   sites <- 1
-  metgrid <- c(1,1,1,1,lon,lat)
+  metgrid <- c(1,1,1,1,floor(lon),floor(lat))
```

Comment:
I probably just accidentally changed it when I was referencing
met2model.ED2.R

On Thu, May 21, 2015 at 3:19 PM, Michael Dietze <notifications@github.com>
wrote:

> In models/ed/R/met2model.ED2.R
> <https://github.com/PecanProject/pecan/pull/490#discussion_r30837209>:
>
> > @@ -267,7 +267,7 @@ for(year in start_year:end_year) {
> >
> >    ## write DRIVER file
> >    sites <- 1
> > -  metgrid <- c(1,1,1,1,lon,lat)
> > +  metgrid <- c(1,1,1,1,floor(lon),floor(lat))
>
> Ann, is there a reason for the change in how the ED met header file is
> written? @robkooper <https://github.com/robkooper> @bcow
> <https://github.com/bcow> what did we determine was the correct setting
> here?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/PecanProject/pecan/pull/490/files#r30837209>.
>